features/product/create_product_and_save_added_attributes.feature

```diff
@@ -29,5 +29,5 @@ Feature: Create product and save a new product value
     And I save the product
     Then the product "gladiator" should have the following values:
       | rate_sale | 0           |
-      | tmp_price | 0.00 EUR    |
+      | tmp_price | 0 EUR       |
```

Comment:
IMO it is more a question of presentation. This step checks the product value data by querying it from the back-end. Not by looking in an exported file or in the UI. This is just the way it is displayed at the moment when you string cast this data.
So in my opinion, no PO concern.